packages/x-codemod/src/util/removeObjectProperty.ts

```diff
@@ -32,7 +32,7 @@ export default function removeObjectProperty({
     })
     .forEach((element) => {
       const targetAttribute = (element.value.openingElement.attributes as any).find(
```

Comment:
IIUC `as any` is turning `attribute` into `any` as well, effectively removing the type safety. Would it be possible to avoid using `as any` in this file and replace with the babel type guards (e.g. `if (t.isIdentifier(...))`)?

Reply:
I tries to remove the `any` of `path.value.openingElement.name as any` and after 30min I still do't have a clue about what are the differences between `JSXIdentifier`, `JSXNamespacedName`, and `JSXMemberExpression`

If someone is interested by this topic, you can find definition of various JSX types here:
https://etclabscore.github.io/json-schema-to-types/globals.html

Reply:
Of all the 3 items mentioned only `JSXMemberExpression` [does not seem to have](https://etclabscore.github.io/json-schema-to-types/interfaces/jsxmemberexpression.html) the `name` property.
So, maybe just do
```suggestion
      const targetAttribute = element.value.openingElement.attributes.filter(attribute => attribute.type !== 'JSXMemberExpression').find(
```
Or just include this case in the find criteria..? 🤔